core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -375,7 +386,16 @@ public static Duration getDuration(PropertyKey key) {
    *         including all default properties
    */
   public static Map<String, String> toMap() {
-    Map<String, String> map = toRawMap();
+    return toMap(false);
+  }
+
+  /**
+   * @param useDisplayValue whether to return display values
+   * @return a view of the resolved properties represented by this configuration,
+   * including all default properties
+   */
+  public static Map<String, String> toMap(boolean useDisplayValue) {
```

Comment:
optional: could use a descriptive name for getting the map with display values like `toDisplayMap`, same for `toRawMap`

Reply:
Done.